material-intro-screen/src/main/java/agency/tango/materialintroscreen/MaterialIntroActivity.java

```diff
@@ -415,37 +415,46 @@ public void onDismissed(Snackbar snackbar, int event) {
         }).show();
     }
 
-    private Integer getBackgroundColor(int position, float positionOffset) {
-        return (Integer) argbEvaluator.evaluate(positionOffset, color(adapter.getItem(position).backgroundColor()), color(adapter.getItem(position + 1).backgroundColor()));
+    private Integer getBackgroundEvaluatedColor(int position, float positionOffset) {
+        return (Integer) argbEvaluator.evaluate(positionOffset, getBackgroundColor(position), getBackgroundColor(position + 1));
     }
 
-    private Integer getButtonsColor(int position, float positionOffset) {
-        return (Integer) argbEvaluator.evaluate(positionOffset, color(adapter.getItem(position).buttonsColor()), color(adapter.getItem(position + 1).buttonsColor()));
+    private Integer getButtonsEvaluatedColor(int position, float positionOffset) {
+        return (Integer) argbEvaluator.evaluate(positionOffset, getButtonsColor(position), getButtonsColor(position + 1));
     }
 
     private int color(@ColorRes int color) {
         return ContextCompat.getColor(this, color);
     }
 
+    private int getButtonsColor(int position) {
+        return color(adapter.getItem(position).buttonsColor());
+    }
+
+    private int getBackgroundColor(int position) {
+        return color(adapter.getItem(position).backgroundColor());
```

Comment:
rename **color** to **getColor** because inside you are calling ContextCompat.getColor